Description: Make test more robust, to pass with Node.js >= 12
Author: Nilesh Patra <npatra974@gmail.com>
Bug: https://github.com/mikolalysenko/uniq/pull/12
Forwarded: https://github.com/mikolalysenko/uniq/pull/12
Last-Update: 2020-01-24

--- a/test/test.js
+++ b/test/test.js
@@ -5,7 +5,7 @@
   t.equals(unique([1,1,2,3,5,5,7]).join(), [1,2,3,5,7].join())
   t.equals(unique([]).join(), [].join())
   t.equals(unique([1,1,1]).join(), [1].join())
-  t.equals(unique([1,1,1,2,2,2], function(a,b) { return (a^b)&1 }).join(), [2,1].join())
+  t.equals(unique([1,1,1,2,2,2], function(a,b) { return (b-a) }).join(), [2,1].join())
 
   t.end()
-})
\ No newline at end of file
+})
